### ISSUE TITLE
EditMOM3 fails to `cei:lem/@wit` and `cei:rdg/@wit`

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#508]
When adding 

```
<cei:app><cei:lem>Apparat</cei:lem><cei:rdg wit="B">Aparratt</cei:rdg></cei:app>
```

with EditMOM3 the Editor offers the attribute `wit` for either `cei:lem` and `cei:rdg` according to the schemas in my/XRX/src/mom/app/cei/xsd.
For validation the attribute would need the `cei` prefix, if the attribute definition is not part of `xs:attributeGroup`.

**Comment by GVogeler** [comment]
Check for other reasons, as https://github.com/GVogeler/mom-ca/commit/e034e7dc0690f9c4f944f5e244a4250b8b20823a introduces the `xs:attributeGroup` but the still occurs.

**Comment by GVogeler** [comment]
solved with #539
